cfg/train_dex-net_4.0_fc_suction.yaml

```diff
@@ -20,7 +20,7 @@ loss: sparse
 optimizer: momentum
 train_l2_regularizer: 0.0005
 base_lr: 0.01
-decay_step_multiplier: 0.5   # number of times to go through training datapoints before stepping down decay rate (in epochs)
+decay_step_multiplier: 0.8   # number of times to go through training datapoints before stepping down decay rate (in epochs)
```

Comment:
Why is this different? Training shouldn't be so sensitive to this, and if it is then let's discuss

Reply:
Fixed.